habitat-lab/habitat/tasks/rearrange/obj_rearrange_sim.py

```diff
@@ -23,3 +30,22 @@ def _setup_targets(self, ep_info):
         self.valid_goal_rec_names = [
             g.object_name for g in ep_info.candidate_goal_receps
         ]
+
+    @property
```

Comment:
Are we using these anywhere? I don't see this being used. We should remove this if it isn't, since this logic is already there in `cat_nav_to_obj_task.py` and this is outdated now as well.

Reply:
Removed in the other PR